### PR TITLE
using better word in persian

### LINKED ISSUE #9: استفاده از نگارش درست فارسی
<p dir=rtl>
پیشنهاد میشه برای بعضی از عبارات از کلمات بهتری استفاده شود.
<br/>
<br/>
برای مثال قسمتی را تغییر دادم.
</p>

### PR DESCRIPTION
**Pull request by mehrdad88sh**
Fix #9

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (amerllica) [inline .idea/workspace.xml:1]: <p dir=rtl>
لطفا این فایل و کلا فولدر idea را از پروژه حذف کنید و در gitignore قرار دهید
</p>

**Review by amerllica — CHANGES_REQUESTED** [review]

**Review by niiima — APPROVED** [review]
It get review and get approved as 'طرح' it's an better alternative for 'پرسش' base on body of the sentence.

**Review by niiima — CHANGES_REQUESTED** [review]
This PR is requesting direct merge into master branch that is not a good practise and might courses merge conflicts.
I am asking for feature pull requests to get ask for dev branch instead of master 
and then merge back into master branch from there.

**Comment by niiima** [comment]
Please use dev branch as a merge target and resend your PR.

**Pull request closed without merging**